package.json

```diff
@@ -23,89 +24,93 @@
     "fast-json-patch": "^3.0.0-0",
-    "file-loader": "^4.0.0",
     "font-awesome": "^4.7.0",
-    "formik": "^2.0.3",
-    "html-webpack-plugin": "^3.2.0",
-    "husky": "^2.3.0",
+    "formik": "^2.0.7",
     "is-url": "^1.2.4",
     "jquery": "^3.4.1",
     "keycloak-js": "^6.0.1",
-    "lint-staged": "^8.1.7",
     "lodash": "^4.17.15",
-    "luxon": "^1.17.3",
-    "mini-css-extract-plugin": "^0.6.0",
-    "node-sass": "^4.12.0",
-    "path": "^0.12.7",
-    "popper.js": "^1.15.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "luxon": "^1.21.3",
+    "popper.js": "^1.16.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
     "react-localization": "^0.1.4",
     "react-resolver": "^3.1.2",
-    "react-router-breadcrumbs-hoc": "^3.2.0",
-    "react-router-dom": "^5.0.0",
-    "react-select": "^3.0.4",
+    "react-router-breadcrumbs-hoc": "^3.2.4",
+    "react-router-dom": "^5.1.2",
+    "react-select": "^3.0.8",
     "react-tagsinput": "^3.19.0",
-    "reactstrap": "^8.0.0",
+    "react-test-renderer": "^16.12.0",
+    "reactstrap": "^8.2.0",
     "recompose": "^0.30.0",
     "sanitize-html": "^1.20.1",
-    "sass-loader": "^7.1.0",
     "showdown": "^1.9.1",
-    "ts-loader": "^6.0.2",
-    "typescript": "^3.4.5",
-    "url-loader": "^1.1.2",
-    "uuid": "^3.3.2",
-    "webpack": "^4.33.0",
-    "webpack-cli": "^3.3.4",
-    "webpack-merge": "^4.2.2",
+    "uuid": "^3.3.3",
     "yup": "^0.27.0"
   },
   "devDependencies": {
-    "@types/enzyme": "^3.9.3",
+    "@babel/register": "^7.7.4",
+    "@svgr/webpack": "^4.3.3",
+    "@types/enzyme": "^3.10.4",
     "@types/enzyme-adapter-react-16": "^1.0.5",
-    "@types/jest": "^24.0.13",
-    "@types/luxon": "^1.15.2",
-    "@typescript-eslint/eslint-plugin": "^2.3.3",
-    "@typescript-eslint/parser": "^2.3.3",
+    "@types/jest": "^24.0.23",
+    "@types/luxon": "^1.21.0",
+    "@types/node": "^12.12.16",
+    "@types/react": "^16.9.16",
+    "@types/react-dom": "^16.9.4",
+    "@types/react-router-dom": "^4.3.5",
+    "@types/react-select": "^3.0.8",
+    "@types/yup": "^0.26.26",
+    "babel-loader": "^8.0.6",
+    "base-href-webpack-plugin": "^2.0.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.1.0",
+    "css-loader": "^3.3.0",
     "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.13.2",
-    "enzyme-to-json": "^3.3.5",
-    "eslint": "^6.5.1",
+    "enzyme-adapter-react-16": "^1.15.1",
+    "enzyme-to-json": "^3.4.3",
+    "file-loader": "^4.3.0",
+    "html-webpack-plugin": "^3.2.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^24.9.0",
+    "lint-staged": "^8.2.1",
+    "mini-css-extract-plugin": "^0.6.0",
+    "node-sass": "^4.13.0",
+    "sass-loader": "^7.3.1",
+    "ts-jest": "^24.2.0",
+    "ts-loader": "^6.2.1",
+    "typescript": "^3.7.3",
+    "url-loader": "^1.1.2",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10",
+    "webpack-merge": "^4.2.2"
+  },
+  "optionalDependencies": {
```

Comment:
I still see no point in optional dependencies, as they only act as confusion

Reply:
ok, jeg skal slå optional og devdependencies sammen.